### PR TITLE
[SE-2281] postgres import

### PR DESCRIPTION
**Pull request by bkochendorfer**
This PR imports our Postgres databases and replica which were created manually in the AWS console. This is phase 1 of this move. Phase 2 is @escattone will be shutting down mysql. I will remove those things from terraform and allow terraform to do the destroy. In Phase 3 I will change the name of this module from `module.mysql` to `module.rds` and do a `terraform state mv`. 

```bash
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # module.mysql-eu-central-1-replica-prod.aws_db_instance.postgres_replica[0] will be updated in-place
  ~ resource "aws_db_instance" "postgres_replica" {
      + apply_immediately                     = true
        id                                    = "mdn-prod-postgres-replica"
      ~ multi_az                              = false -> true
        name                                  = "developer_mozilla_org"
      ~ tags                                  = {
          + "Environment" = "prod"
          + "Name"        = "mdn-prod-postgres-replica"
          + "Region"      = "eu-central-1"
          + "Service"     = "MDN"
          + "Terraform"   = "true"
        }
      ~ tags_all                              = {
          + "Environment" = "prod"
          + "Name"        = "mdn-prod-postgres-replica"
          + "Region"      = "eu-central-1"
          + "Service"     = "MDN"
          + "Terraform"   = "true"
        }
        # (44 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

  # module.mysql-us-west-2.aws_db_instance.mdn_postgres[0] will be updated in-place
  ~ resource "aws_db_instance" "mdn_postgres" {
      + allow_major_version_upgrade           = false
      + final_snapshot_identifier             = "mdn-prod-postgres-final"
        id                                    = "mdn-stage-postgres"
      ~ multi_az                              = false -> true
        name                                  = "developer_allizom_org"
      + password                              = (sensitive value)
      ~ skip_final_snapshot                   = true -> false
      ~ tags                                  = {
          + "Environment" = "stage"
          + "Name"        = "MDN-stage-postgres"
          + "Region"      = "us-west-2"
          + "Service"     = "MDN"
          + "Terraform"   = "true"
        }
      ~ tags_all                              = {
          + "Environment" = "stage"
          + "Name"        = "MDN-stage-postgres"
          + "Region"      = "us-west-2"
          + "Service"     = "MDN"
          + "Terraform"   = "true"
        }
        # (43 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

  # module.mysql-us-west-2-prod.aws_db_instance.mdn_postgres[0] will be updated in-place
  ~ resource "aws_db_instance" "mdn_postgres" {
      + allow_major_version_upgrade           = false
      - deletion_protection                   = true -> null
      + final_snapshot_identifier             = "mdn-prod-postgres-final"
        id                                    = "mdn-prod-postgres"
        name                                  = "developer_mozilla_org"
      + password                              = (sensitive value)
      ~ skip_final_snapshot                   = true -> false
      ~ tags                                  = {
          + "Environment" = "prod"
          + "Name"        = "MDN-prod-postgres"
          + "Region"      = "us-west-2"
          + "Service"     = "MDN"
          + "Terraform"   = "true"
        }
      ~ tags_all                              = {
          + "Environment" = "prod"
          + "Name"        = "MDN-prod-postgres"
          + "Region"      = "us-west-2"
          + "Service"     = "MDN"
          + "Terraform"   = "true"
        }
        # (43 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }
```

### CODE REVIEW

File: apps/mdn/mdn-aws/infra/modules/multi_region/rds/main.tf [hunk 1]
```diff
@@ -82,6 +82,43 @@ resource "aws_db_instance" "mdn_rds" {
...
+  password                     = var.postgres_password
+  publicly_accessible          = false
+  storage_encrypted            = var.postgres_storage_encrypted
+  storage_type                 = var.postgres_storage_type
+  username                     = var.postgres_username
+  performance_insights_enabled = true
+  skip_final_snapshot          = false
+  port                         = var.postgres_port
+  vpc_security_group_ids       = [aws_security_group.mdn_rds_sg[0].id]
+  final_snapshot_identifier    = "mdn-prod-postgres-final"
```
Comment (duallain) [inline apps/mdn/mdn-aws/infra/modules/multi_region/rds/main.tf:118]: should this name for this object be prod for the stage postgres?

**Review by duallain — APPROVED** [review]
generally looks good, one small comment on snapshot naming, but not a big deal either way.

File: apps/mdn/mdn-aws/infra/modules/multi_region/rds/main.tf (hunk 1 above)
Comment (bkochendorfer) [reply apps/mdn/mdn-aws/infra/modules/multi_region/rds/main.tf]: Great :eye: @duallain , pushed up a fix in a3803cc

**Review by escattone — CHANGES_REQUESTED** [review]
Everything looks good to me except for the following (based on the Terraform plan in the introduction):
1. Both `mdn-stage-postgres` and `mdn-prod-postgres-replica` do **not** need `multi-az` to be true, only because it feels like a waste of money.
2. `mdn-prod-postgres` should have `deletion_protection` set to `true`.

**Comment by bkochendorfer** [comment]
@escattone updated the `delete_protection` option. I'd feel more comfortable with `multi-az` set to true as it helps with our availability story. You are correct that it does cost a little bit more but is the same setting we have for mysql so the cost should be the same.

**Review by escattone — APPROVED** [review]
Thanks @bkochendorfer!

**Pull request merged**